src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -558,4 +558,8 @@ describe('Test suite for mappings on existing concepts', () => {
     instance.removeAnswerRow('uniqueKey', true, 'url', 'name', 'code', true);
     expect(spy).toHaveBeenCalled();
   });
+  it('should remove anwser row when a user clicks the remove button while creating a Q-A concept', () => {
```

Comment:
What is the meaning of "anwser"?

Reply:
I have adjusted the descrition, please review